locale/pt_BR/user.xml

```diff
@@ -7,15 +7,28 @@
   * Copyright (c) 2014-2016 Simon Fraser University Library
   * Copyright (c) 2000-2016 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
-  * Locale information: https://pkp.sfu.ca/wiki/index.php?title=OMP:_Portuguese_(pt_BR)
+  * Locale information: https://pkp.sfu.ca/wiki/index.php?title=OJS:_Portuguese_(pt_BR)
```

Comment:
I know... it's just that we (still) don't have a better structure in the wiki with

| locale (general)
|- OJS locale
|- OMP locale

etc. Since this is the latest addition and since it was done for pkp-lib I changed the link to the page with information on these translators...